deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -218,7 +218,7 @@ def run_logic(self, context: Context) -> CheckResult:
 
     def add_condition_drift_score_not_greater_than(self, max_allowed_categorical_score: float = 0.2,
                                                    max_allowed_numeric_score: float = 0.1,
-                                                   number_of_top_features_to_consider: int = 5,
+                                                   num_features_exceeding_threshold: int = 1,
```

Comment:
Should it be "threshold" or "drift_score"? because we call the other inputs "score"

Reply:
maybe add the word "allowed"?

Reply:
Allowed works but will require to change default to 0. (i.e > not >=)